source/features/same-branch-author-commits.tsx

```diff
@@ -16,3 +16,12 @@ void features.add(import.meta.url, {
 	awaitDomReady: true, // Small page
 	init,
 });
+
+/*
+
+Test URLs:
+
+https://github.com/refined-github/refined-github/commits/hotfix
+https://github.com/refined-github/refined-github/commits/main/.github
```

Comment:
The hotfix branch no longer exists, I suggest recreating these URLs on our sandbox repo: https://github.com/refined-github/sandbox